.github/workflows/test-build-outputs.yaml

```diff
@@ -96,3 +96,92 @@ jobs:
             echo $NON_USR_LIB_DYNAMIC_LIBRARIES
             exit 1;
           fi;
+
+  # TODO(devops): extract this into a reusable workflow to be shared with release.yml
```

Comment:
if we were using jsonnet for our workflows, we could easily factorize those things ...